Sources/System/Errno.swift

```diff
@@ -1187,6 +1195,19 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   public static var EBADMACHO: Errno { malformedMachO }
 #endif
 
+#if os(Windows)
+  /// Operation canceled.
+  ///
+  /// The scheduled operation was canceled.
+  ///
+  /// The corresponding C error is `ECANCELED`.
+  @_alwaysEmitIntoClient
+  public static var canceled: Errno { Errno(_ECANCELLED) }
+
+  @_alwaysEmitIntoClient
+  @available(*, unavailable, renamed: "canceled")
+  public static var ECANCELLED: Errno { canceled }
+#else
```

Comment:
[`ECANCELED`][] appears to be supported, possibly by defining [`_CRT_NONSTDC_NO_WARNINGS`][].

Otherwise, does the above comment need to change?

```diff
-  /// The corresponding C error is `ECANCELED`.
+  /// The corresponding C error is `ECANCELLED`.
```

[`ECANCELED`]:                <https://docs.microsoft.com/en-us/cpp/c-runtime-library/errno-constants?view=vs-2019>
[`_CRT_NONSTDC_NO_WARNINGS`]: <https://docs.microsoft.com/en-us/cpp/c-runtime-library/compatibility?view=vs-2019>

Reply:
Thanks for catching that!  That should have been `canceled`.  It doesn't help that I have a tendency to spell it as "cancelled".  I think it is more valuable to have the common spelling here rather than preserve the difference between the spelling that Windows adopts and Unix adopts.